service.ifttt/default.py

```diff
@@ -0,0 +1,112 @@
+import os, sys, re
+import xbmc, xbmcaddon, xbmcgui
+import json, urllib2
+
+__addon__ = xbmcaddon.Addon()
+__addonversion__ = __addon__.getAddonInfo('version')
+__addonid__ = __addon__.getAddonInfo('id')
+__addonname__ = __addon__.getAddonInfo('name')
+__addonPath__ = __addon__.getAddonInfo('path')
+__addonResourcePath__ = xbmc.translatePath(os.path.join(__addonPath__, 'resources', 'lib'))
+__addonIconFile__ = xbmc.translatePath(os.path.join(__addonPath__, 'icon.png'))
+sys.path.append(__addonResourcePath__)
+
+from prefsettings import settings
+
+settings = settings()
+
+LOG_NONE = 0
+LOG_ERROR = 1
+LOG_INFO = 2
+LOG_DEBUG = 3
+
+class Main:
+    def __init__(self):
+        self._init_vars()
+        if (not settings.service_enabled):
+            log(LOG_INFO, "Service not enabled")
+
+        settings.readPrefs()
+        self._daemon()
+
+    def _init_vars( self ):
+        self.Monitor = kodiMonitor()
+        self.Player = kodiPlayer()        
+
+    def _daemon( self ):
+        while (not xbmc.abortRequested):
```

Comment:
this no longer exists in matrix. Please use `xbmc.Monitor.abortRequested()` and `xbmc.Monitor().waitForAbort(secs)` (the later instead of sleep)